packages/flutter_tools/test/web.shard/chrome_test.dart

```diff
@@ -907,6 +907,35 @@ void main() {
     expect(await chromiumLauncher.connect(chrome, false), equals(chrome));
     await chrome.close();
   });
+
+  // testWithoutContext('close is a no-op if it has already been called', () async {
```

Comment:
Do we still need this code?

Reply:
Oops, this was from a previous attempt at fixing this. Thanks for catching!